Author: Alexander Danilov <alexander.a.danilov@gmail.com>
Description: Add new magic for AVCHD Clip Information files (Closes: #538847).

diff -Naurp file.orig/magic/Magdir/images file/magic/Magdir/images
--- file.orig/magic/Magdir/images	2009-12-05 18:09:09.000000000 +0100
+++ file/magic/Magdir/images	2009-12-05 18:10:44.000000000 +0100
@@ -634,3 +634,8 @@
 !:mime	image/x-olympus-orf
 0	string		IIRS		Olympus ORF raw image data, little-endian
 !:mime	image/x-olympus-orf
+
+# Type: files used in modern AVCHD camcoders to store clip information
+# Extension: .cpi
+# From: Alexander Danilov <alexander.a.danilov@gmail.com>
+0	string	HDMV0100	AVCHD Clip Information
